### PR TITLE
Add settings for o2.pl and wp.pl mail providers

### PR DESCRIPTION
**Pull request by disaykin**

### CODE REVIEW

**Comment by dinhviethoa** [comment]
For the other folders, could you add the ones of default? Thanks!

**Comment by disaykin** [comment]
1) O2 provides folder flags, so no need in default folder names in providers.json.

2) It seems that WP.pl does not provide access to the drafts folder via IMAP.

File: resources/providers.json [hunk 1]
```diff
@@ -1278,5 +1278,52 @@
...
+                     }
+                     ]
+        },
+        "domain-match": [
+                         "wp\\.pl"
+                         ],
+        "mailboxes": {
+            "spam": "SPAM",
+            "sentmail": "Wys&AUI-ane",
+            "trash": "Kosz"
```
Comment (dinhviethoa) [inline resources/providers.json]: Could you add those ones here?
```
            "drafts":"Drafts",
            "allmail":"Archive",
```

File: resources/providers.json [hunk 2]
```diff
@@ -1278,5 +1278,52 @@
...
+                     {
+                     "port": 465,
+                     "hostname": "poczta.o2.pl",
+                     "ssl": true
+                     }
+                     ]
+        },
+        "domain-match": [
+                         "o2\\.pl"
+                         ]
```
Comment (dinhviethoa) [inline resources/providers.json]: Could add the mailboxes here anyway?
For consistency, I think there's the fallback for other one too.

**Comment by disaykin** [comment]
Consistency is a good reason. Done.

**Pull request merged**